easybuild/framework/easyblock.py

```diff
@@ -559,7 +561,8 @@ def fetch_extension_sources(self, skip_checksums=False):
 
         return exts_sources
 
-    def obtain_file(self, filename, extension=False, urls=None, download_filename=None, force_download=False):
+    def obtain_file(self, filename, extension=False, urls=None, download_filename=None, force_download=False,
+                    git_config={}):
```

Comment:
@mboisson Using a mutable value like `{}` as default value is evil (see https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).

You have a check `if gitconfig`, so just use `None` as default value?

Reply:
done